lib/pg_saurus/connection_adapters/postgresql_adapter/schema_methods.rb

```diff
@@ -27,7 +27,7 @@ def drop_table(table_name, options = {})
     options     = options.dup
     schema_name = options.delete(:schema)
     table_name  = "#{schema_name}.#{table_name}" if schema_name
-    super(table_name, options)
+    super(table_name, **options)
```

Comment:
Corresponding method in:
- Rails 6.0.0: [`ActiveRecord::ConnectionAdapters::PostgreSQL::SchemaStatements#drop_table`](https://github.com/rails/rails/blob/v6.0.0/activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb#L57)
  - See that the method signature has `options` as a positional argument. Unpacking `options` in the super method call is fine in that case (see example set 3 in the PR description).
- Rails 6.1.0: [`ActiveRecord::ConnectionAdapters::PostgreSQL::SchemaStatements#drop_table`](https://github.com/rails/rails/blob/v6.1.0/activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb#L57)